assets/js/googlesitekit/datastore/site/info.test.js

```diff
@@ -359,5 +359,27 @@ describe( 'core/site site info', () => {
 				expect( referenceURL ).toEqual( baseInfo.referenceSiteURL );
 			} );
 		} );
+
+		describe( 'isSiteURLMatch', () => {
+			beforeEach( async () => {
+				await registry.dispatch( STORE_NAME ).receiveSiteInfo( baseInfo );
+			} );
+
+			it( 'should return TRUE when URL matches the reference site URL even if the protocol is different', () => {
+				expect( registry.select( STORE_NAME ).isSiteURLMatch( 'https://example.com' ) ).toBe( true );
```

Comment:
It would be good to add both here for clarity.
```suggestion
				expect( registry.select( STORE_NAME ).isSiteURLMatch( 'http://example.com' ) ).toBe( true );
				expect( registry.select( STORE_NAME ).isSiteURLMatch( 'https://example.com' ) ).toBe( true );
```